### DISCUSSION TITLE
✅ Add 'View Page' button to the new Editor Top Bar (not to be confused with Preview) - [ED-14270]

### DISCUSSION THREAD

**Discussion by pingram3541** [elementor/elementor#21925] [Feature Request]
### update 2
Learned that pressing ESC to access 'view page' link no longer works. For now I use the CMD+E search interface, not ideal.

<strike>### update
Learned one can press ESC to access 'view page' link but see my comment below about making this capability more known to the end user. Thx</strike>

### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The previous builder interface, which I had zero qualms with, has now removed the daily used feature, "view page".

![Screen Shot 2023-04-04 at 8 36 02 AM](https://user-images.githubusercontent.com/319284/229846663-b81708dd-c523-42c9-bb3e-4e30e60d81be.png)

The makes the process now cumbersome to either use the browser back button, if one entered the builder from the front end admin bar or to take measures to go to the back end and then click through to get to the front end of the page one was editing.

### Describe the solution you'd like

There is a 'Preview' option in the new interface, like it was present in the old, but this is not the proper public facing url, it addes extra processes I don't need to use.

It's nice but I don't use preview, like ever, can we just have the normal 'View Page' option back please?

### Describe alternatives you've considered

Just add a "View Page" option like before. that's it. Thank you.

**Comment by pingram3541** [comment]
Update: As of 8/3/2023 there is no View Page option anymore.

Update: With the help of a suggestion from Nigel Hancock via Chris Ayers (props) in the Facebook Community I've learned when the top bar is enabled (the new builder UI/UE) one can press the ESC button to view the traditional 'navigate from page' options which offers the 'view page' link desired.

I'd like to modify this ticket to request making this ESC function more aware to the end user, ie a better hint it exists. It took me opening this feature request and sharing it in the Facebook Community to learn of it. Thx =)

[![Screen-Shot-2023-04-07-at-10-11-13-PM.png](https://i.postimg.cc/L4JmNGpN/Screen-Shot-2023-04-07-at-10-11-13-PM.png)](https://postimg.cc/rzL6pnzr)

**Reply by loucasl** [reply]
It looks like this has been removed as of 3.13

**Reply by pingram3541** [reply]
Thx, updated original convo.

**Comment by joehdevwp** [comment]
In addition to removing the 'View Page' button they also removed the 'Exit' button. (?!?)

**Reply by pingram3541** [reply]
If you are using the new UI, clicking the Elementor logo in the upper-left brings up a menu with an option to Exit.

[![Screen-Shot-2023-06-28-at-9-27-53-AM.png](https://i.postimg.cc/hvJJnNCv/Screen-Shot-2023-06-28-at-9-27-53-AM.png)](https://postimg.cc/YLH2fDBB)

**Comment by kev200** [comment]
Commenting so this issue will be noticed. I 100% agree with this needing to be added as It's quite literally the weirdest oversights.

**Comment by nicholaszein** [comment]
This request was mentioned in the following threads:

**Discussions:**
- #24491

**Comment by mmg68** [comment]
Please fix this issue

**Comment by mohanadshaban** [comment]
please fix add it again it's so important

**Comment by ashbryant** [comment]
Yep, I thought I was going mad. It had been taken out. Please add it back in

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

![image](https://github.com/elementor/elementor/assets/113993460/d6284af5-e27c-47c9-873d-0d34d4b9fd69)


✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
